cmd/influxd/backup/backup.go

```diff
@@ -229,9 +229,13 @@ func (cmd *Command) backupShard(db, rp, sid string) error {
 		return err
 	}
 
-	cmd.StdoutLogger.Printf("backing up db=%v rp=%v shard=%v to %s since %s",
-		db, rp, sid, shardArchivePath, cmd.since)
-
+	if cmd.isBackup {
+		cmd.StdoutLogger.Printf("backing up db=%v rp=%v shard=%v to %s since %s",
+			db, rp, sid, shardArchivePath, cmd.since)
```

Comment:
Can we format the time appropriately? The default stringer for a `time.Time` will include the monotonic clock offset, if it's set, e.g., something like `m=+0.000416596`. I don't think this is relevant to the user.